.circleci/config.yml

```diff
@@ -270,12 +270,19 @@ commands:
             make
 
   install-xcbeautify-for-xcode14:
```

Comment:
Will this be a problem in hybrids too? 🤔

Reply:
Maybe? Idk if they use xcbeautify. We'll soon see 😅